alloptions-limit.php

```diff
@@ -106,43 +112,30 @@ function wpcom_vip_sanity_check_alloptions_notify( $size, $blocked = false ) {
 		return;
 	}
 
-	$irc_alert_level = 2; // ALERT
-
-	$opsgenie_alert_level = 'P4';
-	if ( $blocked ) {
+	if ( $really_blocked ) {
 		$msg = 'This site is now BLOCKED from loading until option sizes are under control.';
-
-		// If site is blocked, then the IRC alert is CRITICAL
-		$irc_alert_level      = 3;
-		$opsgenie_alert_level = 'P3';
+	} elseif ( $maybe_blocked ) {
+		$msg  = 'This will soon be BLOCKED from loading until if the options sizes increase.';
+		$msg .= PHP_EOL . PHP_EOL;
+		$msg .= sprintf( 'Blocking threshold: %s. Current size: %s', VIP_ALLOPTIONS_ERROR_THRESHOLD, $size_compressed );
 	} else {
 		$msg = 'Site will be blocked from loading if option sizes get too much bigger.';
```

Comment:
I like the extra context that $maybe_blocked is getting in the message, probably useful to give some here as well?

Reply:
Wait, i don't think `$msg` is used anyway. Perhaps remove the whole block?

Reply:
Ah right, that was removed along with the email function.